app/v3/controllers/SubmitForeignPropertyBsasController.scala

```diff
@@ -89,15 +90,15 @@ class SubmitForeignPropertyBsasController @Inject()(val authService: EnrolmentsA
     (errorWrapper.error: @unchecked) match {
       case BadRequestError |
            NinoFormatError |
-           BsasIdFormatError |
            CalculationIdFormatError |
+           ValueFormatError |
```

Comment:
This probably needs to be wrapped in CustomMtdError() as I would expect paths

Reply:
RULE_DUPLICATE_COUNTRY_CODE, and RULE_BOTH_PROPERTIES_SUPPLIED are in the spec but are not included here